tools/runners/run-test-suite.sh

```diff
@@ -44,13 +44,21 @@ then
     shift
 fi
 
-if [ "$1" == "--snapshot" ]
+if [[ "$1" =~ ^--snapshot-tool=.* ]]
 then
     TEST_FILES="$TEST_FILES.snapshot"
     TEST_FAILED="$TEST_FAILED.snapshot"
     TEST_PASSED="$TEST_PASSED.snapshot"
     IS_SNAPSHOT=true;
-    shift
+
+    SNAPSHOT_TOOL=${1#--snapshot-tool=}
+    SNAPSHOT_TOOL=(${SNAPSHOT_TOOL//,/ })
```

Comment:
...following from above: you'd just have `SNAPSHOT_TOOL=$ENGINE-snapshot` here. (agreed, it looses a bit of flexibility by not allowing for snapshot tools placed anywhere in the file system for those who run this script manually. but who does get restricted by that, actually? run-test.py does force to have jerry and jerry-snapshot side-by-side anyway)

Reply:
Certainly we can simply assume the path, but I wanted to give the test site runner more freedom if it is executed manually. 

But I'll update this part.